src/app/data-planes/components/__snapshots__/DataplanePolicies.spec.ts.snap

```diff
@@ -356,15 +356,17 @@ exports[`DataplanePolicies.vue renders snapshot 1`] = `
                         class="cell-origins"
                       >
                         
-                        <div
+                        <ul
                           class="origin-list"
                         >
                           
-                          <a>
-                            fi1.kuma-system
-                          </a>
+                          <li>
+                            <a>
```

Comment:
Sorry I forgot to ask. I'm guessing this (the anchor with no href) is a consequence of us removing router link in the tests right (I vaguely remember thats what we are doing)?

Would be nice if these failed when the routes didn't exist. Would it be a lot of effort to make the snapshots include the `href`'s? (btw I might have misunderstood whats happening here, if so please ignore!)

Reply:
It turns out the test also works without the router stub so I’m removing it. This leads to the snapshot being generated with correct links.